fe/fe-core/src/main/java/com/starrocks/statistic/Constants.java

```diff
@@ -30,4 +30,27 @@ public enum ScheduleStatus {
         FINISH,
     }
 
+    public enum TaskType {
+        MANUAL,
+        PERIODICAL
+    }
+
+    // TaskProcessorType is used to facilitate the generation of specific
+    // executors of TaskRun corresponding to the extension Task
+    public enum TaskProcessorType {
+        SQL
+    }
+
+    // PENDING -> RUNNING -> FAILED
+    //                    -> SUCCESS
+    //                    -> CANCELED
+    //         -> CANCELED
+    public enum TaskRunState {
+        PENDING,
+        RUNNING,
+        CANCELED,
```

Comment:
```suggestion
        CANCELLED,
```

Reply:
CANCELLED and FAILED, keep one of them?

Reply:
Why put these enums in `Constants` file?

Reply:
OK, if the user cancel a TaskRun, We think that this TaskRun has failed, and errMsg is written as canceled by the user. I also tend to reduce the state of the state machine.

I put enums in Constants file because I found AnalyzeType & ScheduleType by statistics module use. I think it's better to put these enums in one place.